app/views/main_layout.scala.html

```diff
@@ -58,6 +59,17 @@
     ))
 }
 
+@content = {
+@contentBlock
```

Comment:
useful to indent this to highlight how page made up of "contentBlock" parameter to layout, and the div you've added